packages/gensim/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: gensim
+  version: 4.2.0
+  top-level:
+    - gensim
+source:
+  url: https://files.pythonhosted.org/packages/24/97/2197f018ee9f8ce2f071b2d9c6711c76159aead710f8d24a2bf006082a28/gensim-4.2.0.tar.gz
+  sha256: 995ebd2970a31d47c100aaac10212f47e2bf12e2b06536d38883c951ff34eef1
+  patches:
+    - patches/builtins.patch
+requirements:
+  run:
+    - numpy
+    - scipy
+    - six
+    - smart_open
+test:
+  imports:
+    - gensim
+    - gensim.corpora
+    - gensim.models
+    - gensim.parsing
+    - gensim.scripts
+    - gensim.similarities
+    - gensim.test
```

Comment:
tests are unvendored by default, so should remove this line, as long as
[`build/unvendor-tests`](https://pyodide.org/en/stable/development/meta-yaml.html#build-unvendor-tests) option is enabled (it is by default).

This is also why CI is failing I think.

Reply:
Thank you, should be better :)